base/applications/mspaint/dialogs.cpp

```diff
@@ -74,6 +74,9 @@ LRESULT CMirrorRotateDialog::OnCancel(WORD wNotifyCode, WORD wID, HWND hWndCtl,
 
 LRESULT CMirrorRotateDialog::OnRadioButton3(WORD wNotifyCode, WORD wID, HWND hWndCtl, BOOL& bHandled)
 {
+    if (IsDlgButtonChecked(IDD_MIRRORROTATERB3) != BST_CHECKED)
```

Comment:
Hmmmm.... How could these OnRadioButtonX be called, to begin with, if these IDD_MIRRORROTATE** radiobutons be not checked (or disabled)?

Reply:
Have a use the Tab key on your keyboard.
That will move the focus.

Reply:
That will generate `BN_SETFOCUS` without checked.